src/arutils.py

```diff
@@ -437,6 +456,8 @@ def polyval2d(x, y, m):
         z += a * x**i * y**j
     return z
 
+def gauss_3deg(x,ampl,cent,sigm):
+    return ampl*np.exp(-1*(cent-x)**2/2/sigm**2)
```

Comment:
Might be a good idea to make the 1 (on the numerator) or 2 (on the denominator) a float. If everything in the exponent is an int (which it shouldn't be, but still), the result would be incorrect.